packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -925,7 +967,11 @@ function updateDynamicChildren(
 
                         // We've already cloned at least once, so it's no longer read-only
                         (oldCh as any[])[idxInOld] = undefined;
-                        insertNode(elmToMove.elm!, parent, oldStartVnode.elm!, renderer);
+                        if (isVFragment(elmToMove)) {
+                            insertFragment(elmToMove, parent, oldStartVnode.elm!, renderer);
+                        } else {
+                            insertNode(elmToMove.elm!, parent, oldStartVnode.elm!, renderer);
+                        }
```

Comment:
These five lines are repeated 3 times... Maybe we could add an `insertFragmentOrNode` utility function?

Reply:
If the argument to that function becomes `VFragment | Node` I think I would need to add a new method on the renderer to be able to check if it's an instance of `Node`, or do some duck-typing to see if it's a `vnode`.

Reply:
I see. OK, well then it's no big deal.